myia/opt/lib.py

```diff
@@ -702,6 +702,15 @@ def replace_applicator(optimizer, node, equiv):
 ##################
 
 
+def check_uses(g):
+    """Returns True if a graph a only one usage."""
```

Comment:
* The function's name could be clearer, e.g. `check_used_once(g)`
* The docstring is not worded correctly.
* I believe this function is equivalent to `sum(mng.graph_users[g].values()) == 1`.

Reply:
* Yes
* Yes
* Probably

Reply:
It is equivalent so I swapped the implementation.